lang/cs/Org.Apache.REEF.Driver/Org.Apache.REEF.Driver.csproj

```diff
@@ -30,7 +30,9 @@ under the License.
   </PropertyGroup>
   <Import Project="$(SolutionDir)\build.props" />
   <ItemGroup>
-    <Reference Include="Microsoft.Hadoop.Avro, Version=1.5.6.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
+    <Reference Include="Microsoft.Hadoop.Avro">
```

Comment:
To unify this entry with all other references (in other projects we reference Avro without version and other parameters)